spyder/plugins/mainmenu/api.py

```diff
@@ -127,9 +127,8 @@ class ProjectsMenuSections:
 
 
 class ToolsMenuSections:
-    Tools = 'tools_section'
-    External = 'external_section'
-    Extras = 'extras_section'
```

Comment:
- Please document these changes in our Changelog, by expanding the entry I added in your previous PR.
- Also, perhaps we should leave the `External` section for external plugins that want to add something to it?

Reply:
* Okay, will do. Wasn't use to it as in the past we'd always leave writing the changelog to right before the release.
* Per our discussion, given this is a brand new unreleased section with no content I don't think it really makes sense to leave arbitrary sections that third party plugins *might* use in the future, when they can just create them themselves if they need it.

Reply:
Fixed in commit [`5e86257` (#24673)](https://github.com/spyder-ide/spyder/pull/24673/commits/5e8625757d44b48e63ba8a39726d13a4dcfc43c4)